export/export.go

```diff
@@ -0,0 +1,427 @@
+	Apply(ctx context.Context, persistentVolumeClaime *v1.PersistentVolumeClaimApplyConfiguration) (*corev1.PersistentVolumeClaim, error)
+}
+
+type StorageClassService interface {
+	List(ctx context.Context) (*storagev1.StorageClassList, error)
+	Apply(ctx context.Context, storageClass *confstoragev1.StorageClassApplyConfiguration) (*storagev1.StorageClass, error)
+}
+
+type PriorityClassService interface {
+	List(ctx context.Context) (*schedulingv1.PriorityClassList, error)
+	Apply(ctx context.Context, priorityClass *schedulingcfgv1.PriorityClassApplyConfiguration) (*schedulingv1.PriorityClass, error)
+}
+
+type SchedulerService interface {
+	GetSchedulerConfig() *v1beta2config.KubeSchedulerConfiguration
+	RestartScheduler(cfg *v1beta2config.KubeSchedulerConfiguration) error
+}
+
+func NewExportService(client clientset.Interface, pods PodService, nodes NodeService, pvs PersistentVolumeService, pvcs PersistentVolumeClaimService, storageClasss StorageClassService, priorityClasss PriorityClassService, schedulers SchedulerService) *Service {
+	return &Service{
+		client:               client,
+		podService:           pods,
+		nodeService:          nodes,
+		pvService:            pvs,
+		pvcService:           pvcs,
+		storageClassService:  storageClasss,
+		priorityclassService: priorityClasss,
+		schedulerService:     schedulers,
+	}
+}
+
+// Get gets all resources from each service.
+func (s *Service) get(ctx context.Context) (*ResourcesForExport, error) {
+	errgrp := util.NewErrGroupWithSemaphore(ctx)
+	resources := ResourcesForExport{}
+
+	if err := s.listPods(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listPods: %w", err)
+	}
+	if err := s.listNodes(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listNodes: %w", err)
+	}
+	if err := s.listPvs(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listPvs: %w", err)
+	}
+	if err := s.listPvcs(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listPvcs: %w", err)
+	}
+	if err := s.listStorageClasses(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listStorageClasses: %w", err)
+	}
+	if err := s.listPcs(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call listPcs: %w", err)
+	}
+	if err := s.getSchedulerConfig(ctx, &resources, &errgrp); err != nil {
+		return nil, xerrors.Errorf("call getSchedulerConfig: %w", err)
+	}
+
+	if err := errgrp.Grp.Wait(); err != nil {
+		return nil, xerrors.Errorf("get resources all: %w", err)
+	}
+	return &resources, nil
+}
+
+func (s *Service) Export(ctx context.Context) (*ResourcesForExport, error) {
+	resources, err := s.get(ctx)
+	if err != nil {
+		return nil, xerrors.Errorf("export resources all: %w", err)
+	}
+	return resources, nil
+}
+
+// Import imports all resources from posted data.
+// (1) Restart scheduler based on the data.
+// (2) Apply each resource to the scheduler.
```

Comment:
Not `to scheduler`, but `to api-server`. I think just deleting `to the scheduler` is fine.

```suggestion
// (2) Apply each resource.
```